src/droplet.F

```diff
@@ -1661,14 +1695,14 @@ subroutine droplet_diag(rtime,nrec,zh,zf,pdata,pdata_locind,dpten,uten,vten,wten
       real :: dumzh(kb:ke),dumzf(kb:ke+1),dzf,xl,yl
       real :: Tpsrc_tmp,qvsrc_tmp,m1src_tmp,m2src_tmp,m3src_tmp
 
-      integer :: tnumpart,tnumdrop,tnumaerosol
-      real :: radavg,radsqr,radmin,radmax,Tpavg,Tpmin,Tpmax,Tpsqr,Tfavg,qfavg,prsavg,rhoavg,tnummult
-      real :: partnum(nk),numconc(nk),vp1mean(nk),vp2mean(nk),vp3mean(nk),vp1msqr(nk),vp2msqr(nk),vp3msqr(nk),Tpsrc(nk),qvsrc(nk),qf(nk),Tf(nk),radmean(nk),radmsqr(nk),Tpmean(nk),Tpmsqr(nk),qfsat(nk),m1src(nk),m2src(nk),m3src(nk)
+      integer :: tnumpart
+      real :: radavg,radsqr,radmin,radmax,Tpavg,Tpmin,Tpmax,Tpsqr,Tfavg,qfavg,prsavg,rhoavg,tnummult,tnumdrop,tnumaerosol
+      real :: partnum(nk),numconc(nk),vp1mean(nk),vp2mean(nk),vp3mean(nk),vp1msqr(nk),vp2msqr(nk),vp3msqr(nk),Tpsrc(nk),qvsrc(nk),qf(nk),Tf(nk),radmean(nk),radmsqr(nk),Tpmean(nk),Tpmsqr(nk),qfsat(nk),m1src(nk),m2src(nk),m3src(nk),qstarmean(nk)
 
       integer, parameter :: nbins = 514   !Number of bins to use for the droplet_diag histograms
-      integer, parameter :: num0 = 12, num1 = 19   !Number of 0th and 1st order statistics in droplet_diag
+      integer, parameter :: num0 = 18, num1 = 20   !Number of 0th and 1st order statistics in droplet_diag
       real,dimension(nbins) :: bins_radius,hist_radius,hist_radius10
```

Comment:
shall we remove these three variables here since they are module variables now?

Reply:
Yes, sorry -- that got missed when fixing the merge conflicts. Note that num0 and num1 need to stay where they are. Those that got moved to the module should be removed (in fact, I don't understand how this compiled for me...)

Reply:
Thanks @drichte2 for the clarification and fix.